recipes/speedy-antlr-tool/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "speedy-antlr-tool" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/speedy-antlr-tool-{{ version }}.tar.gz
+  sha256: ae9de3374dbb867f8b433570a31884ab6eaa0bfdb8c950227eb00ed293983431
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.4
+    - pip
+  run:
+    - python >=3.4
```

Let's stick with what `grayskull` defines here.

```suggestion
    - python >=3.6
    - pip
  run:
    - python >=3.6
```